build/DevDivVsix/ProjectSystemPackage/Microsoft.VisualStudio.ProjectSystem.CommonFiles.swr

```diff
@@ -0,0 +1,21 @@
+use vs
+
+package name=Microsoft.VisualStudio.ProjectSystem.CommonFiles
+        version=2.0.0.0
+        vs.package.chip=neutral
+        vs.package.language=en-us
+
+vs.localizedResources
+  vs.localizedResource language=en-us
+                       title="Microsoft VisualStudio Managed Project System Common Files"
+                       description="Microsoft VisualStudio ProjectSystem for C#/VB(Managed) Projects"
+
+folder "InstallDir:MSBuild\Microsoft\VisualStudio\v15.0\Managed"
+  file source="$(OutputPath)\..\..\src\Targets\Microsoft.CSharp.DesignTime.targets"
```

Comment:
This seems brittle. Can we specify a property for the projectroot in the swixproj and use that here to avoid the relative path between the output and src?